### PR TITLE
setting up the power flow excersise for pandapower

### PR DESCRIPTION
**Pull request by rbolgaryn**
Added the example for pandapower

### CODE REVIEW

**Comment by rbolgaryn** [comment]
Hi @claytonpbarrows , @kdheepak ,

here is my contribution to the example.

Best regards, 
Roman

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/.idea
+/example-pandapower/temp.json
```
Comment (kdheepak) [inline .gitignore:2]: I think you can change the contents of this file to the following:

```suggestion
temp.json
```

and move this file to `example-pandapower/.gitignore`. 

If you want, you can add a new `.gitignore` file to the root level to ignore the `.idea` folder.

**Comment by rbolgaryn** [comment]
Hi @kdheepak ,

can you please check the .gitignore files again?

**Comment by rbolgaryn** [comment]
there are no conflichts @claytonpbarrows

**Review by kdheepak — APPROVED** [review]

**Pull request merged**